src/config.ts

```diff
@@ -273,6 +273,19 @@ export const bankDefinitions: {[contractName: string]: BankInfo} = {
     multi: '0',
     buyLink: null,
   },
+  TheWineryNFT: {
+    name: 'Winery Collection',
+    poolId: 0,
+    sectionInUI: 5,
```

Comment:
Not sure if that's ok to list as 5. 
Since I haven't found any other 5 in the project, I assumed it was ok.